generators/python_functions/most_frequent_words/__init__.py

```diff
@@ -0,0 +1,30 @@
+from pydantic import BaseModel
+from generators.util.spacy import SpacySingleton
+from collections import Counter
+
+INPUT_EXAMPLE = {
+    "text": "APPL went down by 5% in the past two weeks. Shareholders are concerned over the continued recession since APPL and NASDAQ have been hit hard by this recession. Risks pertaining to short-selling are pouring in as APPL continues to depreciate. If the competitors come together and start short-selling, the stock can face calamity.",
```

Comment:
Really like the idea of the module. Extract the most frequent words and then for a search with Google/ Bing to enrich your data quickly. Really simple but powerful! Will probably use this for the next video. :)

Reply:
Running on refinery, it looks so very powerful!!!